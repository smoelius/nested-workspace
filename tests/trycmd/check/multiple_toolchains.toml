args = ["check", "-vv", "--offline"]

[bin]
name = "cargo"

[fs]
cwd = "../../../fixtures/multiple_toolchains"
