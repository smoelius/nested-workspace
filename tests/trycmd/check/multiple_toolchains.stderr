...
[multiple_toolchains 0.1.0] <<< [CWD]/beta
...
[multiple_toolchains 0.1.0]     Finished `dev` profile [unoptimized + debuginfo] target(s) in [..]s
[multiple_toolchains 0.1.0] >>> [CWD]/beta
[multiple_toolchains 0.1.0] <<< [CWD]/nightly
...
[multiple_toolchains 0.1.0]     Finished `dev` profile [unoptimized + debuginfo] target(s) in [..]s
[multiple_toolchains 0.1.0] >>> [CWD]/nightly
...
    Finished `dev` profile [unoptimized + debuginfo] target(s) in [..]s
